src/main/resources/io/viash/platforms/nextflow/VDSL3Helper.nf

```diff
@@ -950,13 +958,10 @@ def workflowFactory(Map args) {
 
         def out = [ output[0], outputFiles ]
 
-        // passthrough additional items
-        if (output[1]) {
-          out.addAll(output[1])
-        }
-
         out
       }
+
+    output_ = output_process.join(output_passthrough)
```

Comment:
@rcannood should I add `failOnDuplicate` here?

Reply:
Yes, good idea!

Reply:
Added!